### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by kylecalbert**

### CODE REVIEW

File: src/components/TicketCards/Card.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { render, screen, fireEvent, getByTestId } from '@testing-library/react';
+import React from 'react';
+import { TicketProvider } from '../../TicketContext/TicketContext';
+import { Card } from './Card';
+import { TicketContext } from '../../TicketContext/TicketContext';
+import { mockData } from '../../test/MockData';
+describe('Testing the Card component', () => {
+  it('Should render the card component', () => {
```
Comment (sukhvir-notta) [inline src/components/TicketCards/Card.test.js]: Rename test as you're testing the delete functionality

File: src/components/Navbar/Navbar.js [hunk 2]
```diff
@@ -47,10 +49,18 @@ export const Navbar = () => {
...
+            <NavbarLink data-cy="home" to="/">
+              Home
+            </NavbarLink>
+            <NavbarLink data-cy="about" to="/About">
+              About
+            </NavbarLink>
+            <NavbarLink data-cy="contact" to="/Contact">
+              Contact
+            </NavbarLink>
+            <NavbarLink data-cy="faq" to="/FAQ">
```
Comment (sukhvir-notta) [inline src/components/Navbar/Navbar.js]: The route path is `/Faq` in App.js which is different to the Link to `/FAQ`. Should these be the same?

File: src/components/Header/HeaderButton.js [hunk 3]
```diff
@@ -14,6 +14,19 @@ import { darkModeColors } from '../../constants/colors';
 import { Text } from '../Text/Text';
 import { useContext } from 'react';
 import { TicketContext } from '../../TicketContext/TicketContext';
+
+//extract function for filter
```
Comment (sukhvir-notta) [inline src/components/Header/HeaderButton.js]: These comments can be removed if not needed in this file

File: cypress/e2e/Navbar.cy.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+describe('Test the Navabr routing works', () => {
+  it('Successfully navigates to the Home Page upon clicking Home', () => {
```
Comment (georgeokez) [inline cypress/e2e/Navbar.cy.js]: Try and use 'should' when starting your bbd tests. You utilised such concepts in the above tests

File: cypress/e2e/Navbar.cy.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    cy.get('[data-cy="about"]').click();
+    cy.url().should('include', '/About');
+  });
+  it('Successfully navigates to the Contact Page upon clicking Contact', () => {
+    cy.visit('http://localhost:3000/');
+    cy.get('[data-cy="contact"]').click();
+    cy.url().should('include', '/Contact');
+  });
+
+  it('Successfully navigates to the FAQ Page upon clicking FAQ', () => {
```
Comment (georgeokez) [inline cypress/e2e/Navbar.cy.js]: Same for all these tests. Start the test names with should. 

E.g should successfully navigate to the FAQ page upon clicking FAQ

File: src/App.js [hunk 6]
```diff
@@ -1,18 +1,29 @@
...
 import { AppContainer } from './App.styles';
-import { Content } from './components/Content/Content';
 import { WebsiteFooter } from './components/Footer/WebsiteFooter';
 import { TicketProvider } from './TicketContext/TicketContext';
+import { Home } from './components/Home/Home';
+import { About } from './components/About/About';
+import { Contact } from './components/Contact/Contact';
+import { Faq } from './components/Faq/Faq';
+
+/////click on button and check url in cypress
```
Comment (georgeokez) [inline src/App.js]: Clean up these comments.

**Pull request merged**